projects/storefrontlib/src/lib/ui/layout/config/default-layout-config.ts

```diff
@@ -19,6 +19,20 @@ export const defaultLayoutConfig: LayoutConfig = {
         'Section5'
       ]
     },
+    CategoryPageTemplate: {
+      slots: ['Section1', 'Section2', 'Section3']
+    },
+    ProductListPageTemplate: {
+      slots: ['ProductListSlot', 'ProductLeftRefinements']
+    },
+    SearchResultsListPageTemplate: {
+      slots: [
+        'Section2',
+        'SearchResultsListSlot',
+        'ProductLeftRefinements',
+        'Section4'
```

Comment:
Is there a purpose that `ProductListPageTemplate` and `SearchResultsListPageTemplate` have different configurations? If I understood correctly, previously they had the same slots or maybe I'm wrong?

Reply:
Previously, the cms slots where not used actually. Now we're adding all potential slots that are used in the backend. Still configurable by customers though.